fele-client-service/utils/auth.js

```diff
@@ -0,0 +1,24 @@
+const authenticateUser = (username, password, organizationName, localOrg) => {
+  let localUser = { "authenticated" : false }
+  const { organization, localUsers } = localOrg;
+
+  if (organization === organizationName) {
+    
+    const localUserIndex = localUsers.findIndex(
+      localUser => localUser.username === username && localUser.password === password
+    );
+    //Found LocalUser
+    if (localUserIndex != -1) {
+      localUser.username = username;
+      localUser.password = password;
```

Comment:
Is there a reason we are returning the password?

Reply:
Good catch. It should not be returned. Will change it